test/Core.Test/Utilities/CoreHelpersTests.cs

```diff
@@ -232,5 +240,187 @@ public void PunyEncode_Success(string text, string expected)
             var actual = CoreHelpers.PunyEncode(text);
             Assert.Equal(expected, actual);
         }
+
+        [Fact]
+        public void GetEmbeddedResourceContentsAsync_Success()
+        {
+            var fileContents = CoreHelpers.GetEmbeddedResourceContentsAsync("data.embeddedResource.txt");
+            Assert.Equal("Contents of embeddedResource.txt\n", fileContents.Replace("\r\n", "\n"));
+        }
+
+        [Theory, CustomAutoData(typeof(UserFixture))]
+        public void BuildIdentityClaims_BaseClaims_Success(User user, bool isPremium)
+        {
+            var expected = new Dictionary<string, string>
+            {
+                { "premium", isPremium ? "true" : "false" },
+                { JwtClaimTypes.Email, user.Email },
+                { JwtClaimTypes.EmailVerified, user.EmailVerified ? "true" : "false" },
+                { JwtClaimTypes.Name, user.Name },
+                { "sstamp", user.SecurityStamp },
+            }.ToList();
+
+            var actual = CoreHelpers.BuildIdentityClaims(user, Array.Empty<CurrentContentOrganization>(),
+                Array.Empty<CurrentContentProvider>(), isPremium);
+
+            foreach (var claim in expected)
+            {
+                Assert.Contains(claim, actual);
+            }
+            Assert.Equal(expected.Count, actual.Count);
+        }
+
+        [Theory, CustomAutoData(typeof(UserFixture))]
+        public void BuildIdentityClaims_OrganizationClaims_Success(User user)
```

Comment:
This test feels a bit unwieldy now that the refactor is completed we should look into adding some regular tests, with input and expected output.

Reply:
Yes, I'm using it to load the migration scripts into EF for MySQL and Postgres migrations. Otherwise, we'd have to have those terrible SQL scripts in a C# file

Reply:
good idea, done